### DISCUSSION TITLE
Back and Forth communication with the bluetooth speaker

### DISCUSSION THREAD

**Discussion by abastola0** [pschatzmann/ESP32-A2DP#427] [Q&A] [answered]
Hi @pschatzmann. Can I, with the means of an esp32 and a Bluetooth speaker, send audio to the speaker and receive the microphone input from the speaker to esp32? Just like answering a phone call through the Bluetooth speaker?

**Comment by pschatzmann** [comment, marked as answer]
I think the Reame is quite clear: __The Hands-Free Profile (HFP), Headset Profile (HSP) and stand alone AVRCP without A2DP are not supported!__

So the answer is still no! You can only do one or the other but not both at the same time...
